app/views/developers/index.html.erb

```diff
@@ -41,6 +41,7 @@
 
     <!-- Paginated list of developers -->
     <div class="col-span-12 md:col-span-9">
+      <%= render Developers::UnseededWarningComponent.new(seedable: Rails.env.development?) %>
```

Comment:
Let's move this to `app/views/shared/_main.html.erb:2` so it shows up on every page.

Reply:
👍